tikv-client/src/main/java/com/pingcap/tikv/Snapshot.java

```diff
@@ -75,6 +76,23 @@ public ByteString get(ByteString key) {
     return client.get(ConcreteBackOffer.newGetBackOff(), key, timestamp.getVersion());
   }
 
+  public void set(ByteString key, ByteString value) {
```

Comment:
should we put `set` in `Snapshot.java`?
`Snapshot` is only for read i think.

Reply:
yeah. I am thinking to refactor this.